src/components/GroupList/GroupItem.js

```diff
@@ -0,0 +1,61 @@
+/* @flow */
+
+import React from 'react'
+import Divider from 'material-ui/Divider'
+import Avatar from 'material-ui/Avatar'
+import FileFolder from 'material-ui/svg-icons/file/folder'
+import { ListItem } from 'material-ui/List'
+import type { GroupType } from 'ducks/projects'
+
+export type Props = {
+  group: GroupType,
+  inset: number,
+  clickHandler: Function,
+}
+
+
+const GroupItem = (props : Props) => {
+  const {
+    group,
+    clickHandler,
+    inset,
+  } = props
+
+  const {
+    name,
+    description,
+    id,
+  } = group
+
+  function touchTapEventHandler() {
+    if (clickHandler) {
```

Comment:
This actually creates a new function every time, so might as well do a `onTouchTap={() => clickHandler(name) }`. For performance reasons this is not that good, but I would rather have pure functions than needing to use React classes where you bind the clickHandler.

Do you agree @kkateq ?